aws-athena/views/default-vw_pin_value.sql

```diff
@@ -11,57 +11,78 @@ WITH stage_values AS (
         parid AS pin,
         taxyr AS year,
         -- Mailed values
-        MAX(
+        ARBITRARY(
+            CASE
+                WHEN
+                    procname = 'CCAOVALUE'
+                    THEN REGEXP_REPLACE(class, '[^[:alnum:]]', '')
+            END
+        ) AS mailed_class,
+        ARBITRARY(
             CASE
                 WHEN procname = 'CCAOVALUE' AND taxyr < '2020' THEN ovrvalasm2
                 WHEN procname = 'CCAOVALUE' AND taxyr >= '2020' THEN valasm2
             END
         ) AS mailed_bldg,
-        MAX(
+        ARBITRARY(
```

Comment:
Thanks @dfsnow for the suggestion.

Reply:
TIL about all the other crazy Trino aggregate functions, like [`max_by()`](https://trino.io/docs/current/functions/aggregate.html#max_by).